notebooks/232-clip-language-saliency-map/232-clip-language-saliency-map.ipynb

```diff
@@ -75,7 +75,7 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.    ov.save_model(text_ov_model, str(text_model_path))
</pre><p>str wrapping not needed anymore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1271/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>